uint8 swarm_id
uint8 sender_id  # robot which has sent the warning
uint8 warn_id    # robot about which the warning has been sent
